hack/generate-internal-groups.sh

```diff
@@ -72,36 +72,36 @@ done
 
 if [ "${GENS}" = "all" ] || grep -qw "deepcopy" <<<"${GENS}"; then
   echo "Generating deepcopy funcs"
-  "${GOROOT}/bin/deepcopy-gen" --input-dirs "$(codegen::join , "${ALL_FQ_APIS[@]}")" -O zz_generated.deepcopy --bounding-dirs "${INT_APIS_PKG},${EXT_APIS_PKG}" "$@"
+  "${GOPATH}/bin/deepcopy-gen" --input-dirs "$(codegen::join , "${ALL_FQ_APIS[@]}")" -O zz_generated.deepcopy --bounding-dirs "${INT_APIS_PKG},${EXT_APIS_PKG}" "$@"
```

Comment:
l think you should fetch `GOPATH` like `go env GOPATH` here because `GOPATH` can be empty here.

Reply:
I think GOPATH shouldn't be empty，but GOBIN maybe empty
it is  almost same to https://github.com/kubernetes/code-generator/blob/master/generate-internal-groups.sh